albumentations/augmentations/transforms.py

```diff
@@ -3001,6 +3002,44 @@ def get_transform_init_args_names(self):
         return "multiplier", "per_channel", "elementwise"
 
 
+class FancyPCA(ImageOnlyTransform):
+    """Augment RGB image using FancyPCA from Krizhevsky's paper
+    "ImageNet Classification with Deep Convolutional Neural Networks"
+
+    Args:
+        alpha (float):  how much to perturb/scale the eigen vecs and vals
+            the paper used std=0.1
```

Comment:
Please add that alpha is selected from the range of `[0, alpha]` using gamma distribution.

Reply:
Thanks, fixed